web/src/main/java/se/inera/intyg/rehabstod/web/controller/api/dto/SickLeavesFilterRequestDTO.java

```diff
@@ -21,22 +21,23 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import lombok.Data;
+import se.inera.intyg.rehabstod.service.diagnos.dto.DiagnosKapitel;
 
+@Data
 public class SickLeavesFilterRequestDTO {
     private List<String> doctorIds;
-    private int toSickLeaveLength;
-    private int fromSickLeaveLength;
+    private Integer toSickLeaveLength;
+    private Integer fromSickLeaveLength;
+    private List<DiagnosKapitel> diagnosisChapters;
 
-    public SickLeavesFilterRequestDTO() { }
```

Comment:
For serializing to work, we need to keep our default constructor. Otherwise we will get an error.